library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -216,12 +249,23 @@ SAMLCore_3_4_1_1c=When a Format defined in Section 8.3 other than \
   value of the <NameID> within the <Subject> of any <Assertion> MUST be identical to the \
   SPNameQualifier value supplied in the <NameIDPolicy>.
 
-SAMLCore_3_4_1_4a=The responder MUST ultimately reply to an <AuthnRequest> with a <Response> \
+SAMLCore_3_4_1_4_a=The responder MUST ultimately reply to an <AuthnRequest> with a <Response> \
   message containing one or more assertions that meet the specifications defined by the request
-SAMLCore_3_4_1_4b=At least one statement in at least one assertion MUST be a <saml:AuthnStatement> \
+
+SAMLCore_3_4_1_4_b=If the <saml:Subject> element in the request is present, then the resulting \
+  assertions' <saml:Subject> MUST strongly match the request <saml:Subject>, as described in \
+  Section 3.3.4, except that the identifier MAY be in a different format if specified by \
+  <NameIDPolicy>.
+
+SAMLCore_3_4_1_4_b1=If the <saml:Subject> element in the request is present, then the resulting \
+  assertions' <saml:Subject> MUST strongly match the request <saml:Subject>, as described in \
+  Section 3.3.4
```

Comment:
❗️ I don't like breaking spec references up like this so that the messaging can be more specific. I don't like it for a few reasons:
1. A more targeted message is what the `message` property in the `SamlComplianceException` is for.
2. We end up with the overall messages in here which are not being used at all.
3. It adds more complexity to understanding the code as a developer.